--- a/ecm_interface.c
+++ b/ecm_interface.c
@@ -4584,17 +4584,17 @@
 		if (skb && (skb->skb_iif == dev->ifindex)) {
 			struct pppol2tp_common_addr info;

-			if (__ppp_is_multilink(dev) > 0) {
+			if (ppp_is_multilink(dev) > 0) {
 				DEBUG_TRACE("%px: Net device: %px is MULTILINK PPP - Unknown to the ECM\n", feci, dev);
 				type_info.unknown.os_specific_ident = dev_interface_num;

 				/*
 				 * Establish this type of interface
 				 */
 				ii = ecm_interface_unknown_interface_establish(&type_info.unknown, dev_name, dev_interface_num, ae_interface_num, dev_mtu);
 				return ii;
 			}
-			channel_count = __ppp_hold_channels(dev, ppp_chan, 1);
+			channel_count = ppp_hold_channels(dev, ppp_chan, 1);
 			if (channel_count != 1) {
 				DEBUG_TRACE("%px: Net device: %px PPP has %d channels - ECM cannot handle this (interface becomes Unknown type)\n",
 					    feci, dev, channel_count);
@@ -4699,25 +4699,25 @@
 	 * PPP - but what is the channel type?
 	 * First: If this is multi-link then we do not support it
 	 */
-	if (__ppp_is_multilink(dev) > 0) {
+	if (ppp_is_multilink(dev) > 0) {
 		DEBUG_TRACE("%px: Net device: %px is MULTILINK PPP - Unknown to the ECM\n", feci, dev);
 		type_info.unknown.os_specific_ident = dev_interface_num;

 		/*
 		 * Establish this type of interface
 		 */
 		ii = ecm_interface_unknown_interface_establish(&type_info.unknown, dev_name, dev_interface_num, ae_interface_num, dev_mtu);
 		return ii;
 	}

 	DEBUG_TRACE("%px: Net device: %px is PPP\n", feci, dev);

 	/*
 	 * Get the PPP channel and then enquire what kind of channel it is
 	 * NOTE: Not multilink so only one channel to get.
 	 */
-	channel_count = __ppp_hold_channels(dev, ppp_chan, 1);
+	channel_count = ppp_hold_channels(dev, ppp_chan, 1);
 	if (channel_count != 1) {
 		DEBUG_TRACE("%px: Net device: %px PPP has %d channels - ECM cannot handle this (interface becomes Unknown type)\n",
 				feci, dev, channel_count);
 		type_info.unknown.os_specific_ident = dev_interface_num;
@@ -10833,12 +10833,12 @@
 			/*
 			 * Get the PPP channel to extract L2TP tunnel information
 			 */
-			if (__ppp_is_multilink(dev) > 0) {
+			if (ppp_is_multilink(dev) > 0) {
 				DEBUG_TRACE("%px: Net device: %px is MULTILINK PPP - Not supported\n", skb, dev);
 				return false;
 			}

-			channel_count = __ppp_hold_channels(dev, ppp_chan, 1);
+			channel_count = ppp_hold_channels(dev, ppp_chan, 1);
 			if (channel_count != 1) {
 				DEBUG_TRACE("%px: Net device: %px PPP has %d channels - Not supported\n", skb, dev, channel_count);
 				return false;
 			}
